src/App.asm

```diff
@@ -1,6 +1,61 @@
-.data
-
+.data	
+str1: .ascii "Enter total number of students: \0"
+str2: .ascii "Enter student score: \0"
+str3: .ascii "Highest score is \0"
+count: .word 0
+i: .word 0
+top_student: .word-1
+score: .word 0
 
 .globl main
 
-.text
\ No newline at end of file
+
+.text
+
+
+main:
+li $t6,1
+lw $t4,top_student
+
+li $v0,4
+la $a0,str1
+syscall
+
+li $v0,5
+syscall
+move $t0,$v0
+sw $t0, count
+
+lw $t1,i
+
+loop_begin:
+blt $t1,$t0,loop_body
```

Comment:
before while loop condition, variables `i` and `count` should be loaded from memory such as:
```asm
...
lw  $t1, i
lw  $t0, count
blt  $t1, $t0, loop_body
...
```